snuba/datasets/entity_subscriptions/pluggable_entity_subscription.py

```diff
@@ -1,15 +1,30 @@
 from typing import Sequence
 
-from snuba.datasets.entity_subscriptions.entity_subscription import SessionsSubscription
+from snuba.datasets.entity_subscriptions.entity_subscription import (
+    ComplexEntitySubscription,
+    SimpleEntitySubscription,
+)
 
 
-class PluggableEntitySubscription(SessionsSubscription):
+class PluggableEntitySubscriptionComplex(ComplexEntitySubscription):
```

Comment:
Nit: I don't think `Simple` and `Complex` are the right terms. I would leave the `complex` one just as `PluggableEntitySubscription` and add `BasicPluggableEntitySubscription`.